test/libsolidity/syntaxTests/freeFunctions/free_function_visibility.sol

```diff
@@ -0,0 +1,4 @@
+function fun() public {
+}
+// ----
+// SyntaxError 4126: (0-25): Free functions cannot have visibility.
```

Comment:
Does it say the same if you give `internal`?

Reply:
Yes, will add another function with internal visibility.